docs/step-by-step/kubeapps-on-tkg/step-2.md

```diff
@@ -203,21 +211,13 @@ roleRef:
 subjects:
   - apiGroup: rbac.authorization.k8s.io
     kind: User
-    name: email@example.com
+    name: USERNAME
```

Comment:
Check this

Reply:
AFAIK, it has to be a USER_EMAIL rather than a USERNAME (since the oidc provider just gives us the email as the user identifier). But yep, I like the capitalized approach to make it clear the things to be repaced